lib/kotkowo_web/live_components/cat_filter.ex

```diff
@@ -122,7 +122,7 @@ defmodule KotkowoWeb.LiveComponents.CatFilter do
             <div class="flex flex-col space-y-4">
               <h3 class="font-bold text-primary text-sm">Imię</h3>
               <input
-                phx-debounce
+                phx-debounce={100}
```

Comment:
why 100?
it's 300 by default. don't see a reason to change this

Reply:
300 feels way to laggy, besides it's set to ="100" everywhere else